docs/markers.md

```diff
@@ -183,3 +183,65 @@ resource](workload-collections.md#collection-resources) it will be treated as a
 collection marker and will configure a field in the collection's custom
 resource.
 
+## Resource Marker
+
+Defined as `+operator-builder:resource` this marker can be used to control a specific
+resource with arguments in the marker.
+
+### Arguments
```

Comment:
This Arguments section looks like a copy-paste from the earlier section on Field Markers.  Perhaps a general explanation of arguments belongs at the top as it applies to both Field Markers and Resource Markers.  Then each section can just list the supported arguments for each marker.